crates/issue/src/lib.rs

```diff
@@ -316,11 +321,8 @@ impl<T: Config> Pallet<T> {
         ext::vault_registry::ensure_not_banned::<T>(&vault_id)?;
 
         // calculate griefing collateral based on the total amount of tokens to be issued
-        let amount_collateral = amount_requested.convert_to(T::GetGriefingCollateralCurrencyId::get())?;
```

Comment:
Shouldn't you also remove the `GetGriefingCollateralCurrencyId` associated type from the vault-registry? It's supposed to not be used at all any longer, and that will help point out if other parts of the code should also be changed

Reply:
I noticed that `nomination`, `redeem` and `replace` still use it and it's no longer present in the `issue` pallet, so I guess it's fine. Would be cleaner to have the same griefing collateral everywhere though, but that's a product decision.

Reply:
redeem doesn't use it (except for an interaction with to-be-replaced tokens). Replace does use it, and we can change that in the future if we want, but not a prio for now